src/pages/Swap/index.tsx

```diff
@@ -65,6 +74,18 @@ const AppBodyContainer = styled.section`
   align-items: center;
   z-index: 3;
   min-height: calc(100vh - 340px);
+  width: 100%;
+  max-width: 460px;
```

Comment:
`width` is unnecessary when you use `max-width`

Reply:
Yes it is,Fixed. Thaaaanks.